resources/views/layouts/app.blade.php

```diff
@@ -49,12 +51,26 @@
                                 </li>
                             @endif
                         @else
+
+                            <li class="nav-item">
+                                <a class="nav-link" href="/home">Home</a>
```

Comment:
I'd be better if you use URL::route() instead for href link.

Reply:
implemented!